src/docs/markdown/running.md

```diff
@@ -152,6 +153,18 @@ RestartSec=5s
 Then, save the file and exit the text editor, and restart the service for it to take effect:
 <pre><code class="cmd bash">sudo systemctl restart caddy</code></pre>
 
+### SELinux System Considerations
+
+On SELinux enabled systems, systemd unit files and their executables will not be run unless labelled with `systemd_unit_file_t` and `bin_t` respectively.
```

Comment:
Can/should we add a note that if installing Caddy from the COPRs, this is taken care of for you?